pkg/docker/docker.go

```diff
@@ -472,17 +479,40 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) (err error) {
 		}
 	}
 
-	glog.V(2).Infof("Waiting for container")
-	exitCode, err := d.client.WaitContainer(container.ID)
-	wg.Wait()
-	if err != nil {
-		return err
-	}
-	glog.V(2).Infof("Container exited")
+	if opts.TargetImage {
+		exposedPorts := make([]string, 0)
+		for port := range imageMetadata.Config.ExposedPorts {
+			exposedPorts = append(exposedPorts, port.Port())
+		}
 
-	if exitCode != 0 {
-		return errors.NewContainerError(container.Name, exitCode, "")
+		glog.V(0).Infof("\n\n\n\n\nThe image %s has been started in container %s as a result of the --run=true option.  The container's stdout/stderr will be redirected to this command's glog output to help you validate its behavior.  You can also inspect the container with docker commands if you like.  If the container is set up to stay running, you will have to Ctrl-C to exit this command, which should also stop the container %s.  This particular invocation attempts to run on the exposed ports %+v \n\n\n\n\n", image, container.ID, container.ID, exposedPorts)
```

Comment:
I was thinking you'd print the port mappings (ie the host port and what container port it mapped to).  Just the internal port numbers aren't too useful here.